src/stakingrewards/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "stakingrewards",
```

Comment:
Will this package be published under `threshold` in Node, so something like `threshold/stakingrewards`?

Reply:
I think it is not necessary to publish the package in NPM at this time since these functions are used only in this repo scripts (i.e gen_merkle_dist.js) and not in external repos